controller/src/main/java/com/epam/esm/util/ResponseAssembler.java

```diff
@@ -138,6 +138,10 @@ private static TagDTO mapTag(TagDTO tagDTO) {
     }
 
     private static UserDTO mapUser(UserDTO userDTO) {
+        if (!ObjectUtils.isEmpty(userDTO.getOrders())) {
```

Comment:
What's about `org.apache.commons.lang3.ObjectUtils#isNotEmpty` method?

Reply:
I believe they have the same functionality, but i changed to the apache class